src/app/helpers/direction.helper.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { Direction } from '../enums/direction.enum';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class DirectionHelper {
+    getDirection(key: string): Direction {
+        switch (key) {
+            case 'ArrowLeft':
+                return Direction.LEFT;
+            case 'ArrowUp':
+                return Direction.UP;
+            case 'ArrowDown':
+                return Direction.DOWN;
+            case 'ArrowRight':
+            default:
+                return Direction.RIGHT;
+        }
+    }
+
+    getOppositeDirection(direction: Direction): Direction {
+        switch (direction) {
+            case Direction.LEFT:
+                return Direction.RIGHT;
+            case Direction.RIGHT:
+                return Direction.LEFT;
+            case Direction.UP:
+                return Direction.DOWN;
+            case Direction.DOWN:
+            default:
+                return Direction.UP;
```

Comment:
Maybe we should default to something like null or undefined instead. We use this function's results for some comparisons, so it wouldn't be an issue, would even be a better solution

Reply:
FYI #9 updates the getDirection method.
I agree that the default should be null/undefined.